collect-adb.rst

```diff
@@ -0,0 +1,115 @@
+*****************************************
+Using Android Debug Bridge with Collect
+*****************************************
+
+This document focuses specifically on using **adb** as a command line tool to perform tasks related to Collect. 
+
+.. _what-is-adb:
+
+What is Android Debug Bridge (adb)?
+====================================
+
```

Comment:
If you cut lines 5-11, I think nothing is lost.

It is *usually* not necessary to say what a doc is about, especially if the title is descriptive enough. Then you can just get right to answering the question "What is adb?" without needing a second headline.